source/components/atoms/Input/Input.tsx

```diff
@@ -58,6 +70,7 @@ const Input: React.FC<InputProps> = React.forwardRef(
           onSubmitEditing={() => {
             Keyboard.dismiss();
           }}
+          keyboardType={inputType ? keyboardTypes[inputType] : keyboardType}
```

Comment:
Define check in const and pass the const to the component keyboardType prop

Reply:
Fixed.